### ISSUE TITLE
The code of IBQ loss is mismatch with the paper eq10.

### ISSUE THREAD

**Issue by yufan-aslp** [TencentARC/SEED-Voken#50]
`quant_loss = torch.mean((z_q - z)**2) + torch.mean((z_q_2.detach()-z)**2) + self.beta * \
                   torch.mean((z_q_2 - z.detach()) ** 2)`

this code is at src/IBQ/modules/vqvae/quantize.py 453

But in the paper eq10 the beta is for the z with the z_q_2.detach()
